### ISSUE TITLE
Small fixes in Nausicaa

### ISSUE THREAD

**Issue by trynan** [nedragrevev/custom-subs#6]
Hey there, thanks so much for all the work you've done! I noticed a couple small things when watching Nausicaa and figured I'd point them out.

- Line 3670: "Why'd they bring this bundle of trouble here."
   I thought this should be a question mark instead of a period.
- Line 3771: "Even our largest ship couldn't \Nsupport it's weight and crashed."
   it's --> its.
- Line 3917: "The engine's are too noisy. Hurry!"
   engine's --> engines.

That's all I noticed. Thanks again!

**Comment by nedragrevev** [comment]
Thanks so much for letting me know about those errors (and noticing them, too).

I've applied all those fixes and made a few more grammatical adjustments (a quick search all in notepad++ showed some commas needing removal, and some inconsistency with interrobang).

While doing that, I came across a somewhat unnatural line @ ~1h27m52s (line 4313), so I've given it a more natural translation.

**Comment by nedragrevev** [comment]
Thanks again.

**Comment by nedragrevev** [comment]
I wanted to let you know that I've just completed a full dialogue check with the Nausicaa subs.
I ended up finding 7 more errors: 5 were missing commas that were required; 2 were missing "'s" contractions.
I've made some additional improvements for various lines (naturalness) and retimed one line which overlapped with another.

If you're interested, feel free to re-download the update. I'm confident it'll be the final release because the method of checking I used is far, far more reliable than anything I did in the past.

And I have to thank you again because it was your feedback that motivated me to check just about every release (and I've spotted mistakes in all the ones I've checked so far). I didn't know about checking the dialogue in a text editor (reading it like a novel) until your comment, and in terms of its reliability to identify errors, even with really minor stuff, there is just no comparison to trying to QC by watching or going line-by-line in Aegisub (too many things to do at once if the video and audio are playing). It's quicker than watching it through or editing in Aegisub too. I just wish I learned about/recognized the value of this method of error checking when I started subbing; it would've saved me all these adjustments.
